source/openstack/index.rst

```diff
@@ -0,0 +1,16 @@
+.. _openstack:
+
+OpenStack (x86_64, POWER, & AARCH64)
+====================================
+
+This wiki provides resources to deploy and manage instances on our OpenStack clusters.
+
+We curently have 3 OpenStack clusters at the OSL: x86_64, POWER, and AARCH64. The x86 cluster is used
+typically for production hosting while the other two are mainly used to support development and CI
+for those architectures. All 3 clusters run OpenStack powered by KVM.
```

Comment:
I think it would be good here to provide a list displaying the URL's for each cluster.